index.html

```diff
@@ -56,5 +56,9 @@
     </div>
   </div>
   <!-- built files will be auto injected -->
+  <noscript>
+    <h3>We're sorry but this site requires a browser that supports JavaScript.</h3>
```

Comment:
JS can not work not only case when the browser does not support it. The user can turn off JS manually. So, as I think, we need to change the message to something more suitable.